src/Microsoft.Extensions.ML/Builder/BuilderExtensions.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.ML
+{
+    /// <summary>
+    /// Extension methods for <see cref="PredictionEnginePoolBuilder{TData, TPrediction}"/>.
+    /// </summary>
+    public static class BuilderExtensions
+    {
+        /// <summary>
+        /// Adds the model at the specified location to the builder.
+        /// </summary>
+        /// <param name="builder">The builder to which to add the model.</param>
+        /// <param name="uri">The location of the model.</param>
+        /// <returns>
+        /// The updated <see cref="PredictionEnginePoolBuilder{TData, TPrediction}"/>.
+        /// </returns>
```

Comment:
Is this a public C# API? Would it need a sample file? The same question to the subsequent functions.

Reply:
The APIs in this project are part of the public API. We have samples using them here:

https://github.com/dotnet/machinelearning-samples/tree/1131a9eb49f45cd0ff96e444ec584c78e77dc6c6/samples/csharp/end-to-end-apps/ScalableMLModelOnWebAPI-IntegrationPkg

These APIs are different from the "core" APIs in that they don't transform data or learn from data. But instead they are intended for bridging ML.NET and ASP.NET (and similar) technologies. So the same "sample" format for transforms and trainers won't work. We need a full-blown sample for using it end-to-end, like we have in machinelearning-sample.